### PR TITLE
Test #659

### PR DESCRIPTION
**Pull request by joseph-robertson**
Resolves #[issue number here].

## Pull Request Description

[description here]

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [ ] All rake tasks have been run, and pass
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [ ] All tests are passing
- [ ] The [changelog](https://github.com/NREL/resstock/blob/develop/CHANGELOG.md) has been updated appropriately
- [ ] This branch is up-to-date with develop

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

File: measures/BuildExistingModel/measure.rb [hunk 1]
```diff
@@ -28,7 +28,7 @@ def name
 
   # human readable description
   def description
-    return 'Builds the OpenStudio Model for an existing building.'
+    return 'Builds the OpenStudio Model for a existing building.'
```
Comment (shorowit) [inline measures/BuildExistingModel/measure.rb:31]: Why are we replacing proper grammar with improper grammar?

Comment (joseph-robertson) [reply measures/BuildExistingModel/measure.rb:31]: We isn't. It's a temporary change to see if results_output.csv changes. :)

**Comment by joseph-robertson** [comment]
@shorowit Weird. Didn't work. Any ideas other than sorting `model.getFoo` arrays?

**Comment by shorowit** [comment]
Yes, I can think of several additional reasons why the order of things can change; I just gave the `model.getFooObjects` as one example. I don't think it's worth the effort of trying to address this, personally. It'll get fixed when restructure takes over.

**Comment by joseph-robertson** [comment]
Oh. I guess I thought it would be pretty easy.

**Pull request closed without merging**